DNN Platform/Library/Security/Membership/AspNetMembershipProvider.cs

```diff
@@ -1794,7 +1791,7 @@ public override void UpdateUsersOnline(Hashtable userList)
                     }
                     else
                     {
-                        loginStatus = UserLoginStatus.LOGIN_USERLOCKEDOUT;
+                        loginStatus = UserLoginStatus.LOGIN_USERLOCKEDOUT;                      
                     }
```

Comment:
DataCache.ClearUserCache(portalId, username);
this is also working if I put it here. So What I'm thinking that put back those at the beginning and one here

Reply:
@Mohtshum Yes, I think it should, but I don't believe that we can guarantee that to be the case, login should ALWAYS be pulling a non-cached user.  So It think re-adding the ones at the beginning and then one here would be the best option to do this quickly

Reply:
Did revert and put the cache clear call after Lockedout and approval call